proposals/MultipleReadersWriters.md

```diff
@@ -221,6 +221,11 @@ childAccessHandle.close();
 await parentHandle.remove(); // successful
 ```
 
+### Interactions with BFCache
+A page should not be made aware that it or another page is or has been in the BFCache. A fully active page could be made aware of a BFCached page if there is contention between locks they hold. The File System Access API operations/primitives hold locks that may cause this issue.
+
+To keep BFCache enabled when a site uses the File System Access API, a BFCached page should be evicted only on locking contention with a fully active page. This allows the site to have the performance gains of BFCache up until it would be made aware of the BFCache.
```

Comment:
> a BFCached page should be evicted only on locking contention with a fully active page

A BFCached page may be evicted for any number of reasons unrelated to this API. Rather than say that a page will _only_ be evicted due to lock contention (which implies that no lock contention == no eviction), we should say that a page _must_ be evicted if it causes lock contention (which says nothing about what happens when there's no lock contention)

Reply:
Ok updated